### PR TITLE
Get-CSTeamsShiftsConnectionInstance and Test-CSTeamsShiftsConnectionValidate docs updated

### PR DESCRIPTION
**Pull request by leonardospina**
Client relatable url removed from example in Get-CSTeamsShiftsConnectionInstance
New syntax updated in Test-CSTeamsShiftsConnectionValidate (missed in previous PR)

### CODE REVIEW

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

File: teams/teams-ps/teams/Test-CsTeamsShiftsConnectionValidate.md [hunk 1]
```diff
@@ -71,127 +81,14 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -ConnectorSpecificSettingLoginUserName
-
-The login user name to the WFM team.
+### -ConnectorSpecificSettings
+The connector specific settings
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Test-CsTeamsShiftsConnectionValidate.md:85]:
```suggestion
The connector specific settings.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @leonardospina, kindly commit the minor suggested change. Many thanks.

**Comment by leonardospina** [comment]
@JohanFreelancer9 done, thanks for reviewing!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tseward Gentle reminder please.

File: teams/teams-ps/teams/Test-CsTeamsShiftsConnectionValidate.md [hunk 2]
```diff
@@ -32,7 +32,17 @@ This cmdlet validates Workforce management (WFM) connection settings. It validat
...
+PS C:\> Test-CsTeamsShiftsConnectionValidate -ConnectorId "6A51B888-FF44-4FEA-82E1-839401E00000" -ConnectorSpecificSettings (New-Object Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.ConnectorSpecificBlueYonderSettingsRequest -Property @{ AdminApiUrl = "https://contoso.com/retail/data/wfmadmin/api/v1-beta3"; SiteManagerUrl = "https://contoso.com/retail/data/wfmsm/api/v1-beta4"; EssApiUrl = "https://contoso.com/retail/data/wfmess/api/v1-beta2"; RetailWebApiUrl = "https://contoso.com/retail/data/retailwebapi/api/v1"; CookieAuthUrl = "https://contoso.com/retail/data/login"; FederatedAuthUrl = "https://contoso.com/retail/data/login"; LoginUserName = "PlaceholderForUsername"; LoginPwd = "PlaceholderForPassword" }) -Name $InstanceName
```
Comment (tseward) [inline teams/teams-ps/teams/Test-CsTeamsShiftsConnectionValidate.md:35]: Is there more documentation about the AdminApiUrl/SiteManagerApiUrl/EssApiUrl and where one would find that? I see they are marked as betas in the examples. Also, while I see a CID/Secret, what are the LoginUserName/LoginPwd used for and will this cmdlet run with MFA enabled?

**Comment by yogkumgit** [comment]
@leonardospina Please comment. Thanks

File: teams/teams-ps/teams/Test-CsTeamsShiftsConnectionValidate.md (hunk 2 above)
Comment (leonardospina) [reply teams/teams-ps/teams/Test-CsTeamsShiftsConnectionValidate.md:35]: All those values depend on customer managed third party systems so the actual values depend on their own installations.
Docs are here, still under review atm:
https://review.docs.microsoft.com/en-us/microsoft-365/frontline/shifts-connectors?view=o365-worldwide&branch=v-lanachin-ukg

**Review by tseward — APPROVED** [review]

**Pull request merged**